pydomcfg/utils.py

```diff
@@ -26,6 +27,51 @@ def is_nemo_none(var: Optional[float] = None) -> bool:
     return var in [None, 999999.0]
 
 
+# -----------------------------------------------------------------------------
+def calc_rmax(depth: DataArray) -> DataArray:
```

Comment:
Same used by the `Bathymetry` class, right?
Then we could delete it in `tests/bathymetry.py` and import from here.